packages/frontend/src/creation-form/index.tsx

```diff
@@ -335,23 +264,6 @@ export const Component = ({
                             />
                         )}
                     </StepCard>
-                    <StepCard
-                        title={t("card.outcome.configuration.title")}
-                        step={enableOraclePickStep ? 5 : 4}
```

Comment:
Since we removed the step, we have to subtract `1` from the other steps' `step`  property.

Reply:
Oh you're right. Addressed in 59d383882e08905bbd97b99a840f63d3f2626170